pandas/core/base.py

```diff
@@ -1185,6 +1185,10 @@ def drop_duplicates(self, keep='first', inplace=False):
               occurrence.
             - False : Mark all duplicates as ``True``.
         take_last : deprecated
+        shortcircuit : boolean, False
+            return True if ANY value is duplicated, False otherwise
```

Comment:
What do you mean with 'evaluate the entire space'? Both have to look at all values of the index or series?